data/tools/GUI.pyw

```diff
@@ -56,38 +56,39 @@ WESNOTH_TRAN_DIR = os.path.join(WESNOTH_ROOT_DIR, "translations")
 _ = lambda x: x
 
 
-def set_default_locale():
+def set_user_locale():
     global _
 
     # TODO: Replace CLI args for a proper locale selection GUI.
     # More importantly, code to dynamically update the text/layout is missing.
     parser = argparse.ArgumentParser(
-        description=_("Open a Graphical User Interface (GUI) to WML Tools"),
-        usage=("GUI.pyw [--lang=" + _("LANGUAGE") + "]")
+        description=_("Open a Graphical User Interface (GUI) to WML Tools")
     )
     parser.add_argument(
         '--lang',
-        # TRANSLATORS: ISO 15897 "Procedures for the registration of cultural elements" specifies use
-        # of underscores in the locale id. For example, use en_GB, not en-GB.
-        help=_("Launch GUI.pyw in the given language. Language code must follow ISO 15897 (e.g. en_GB).")
+        help=_("Launch GUI.pyw in the specified language. Language code is expected as a POSIX locale name, refer to "
+               "gettext.wesnoth.org for a full list.")
     )
+
     opts = parser.parse_args(sys.argv[1:])
     if opts.lang is not None:
         if not os.path.isdir(WESNOTH_TRAN_DIR):
-            showerror(_("Error"),
-                      # TRANSLATORS: {0} is "translations", the folder where compiled translation files (.mo) are stored.
-                      _("`{0}` folder not found. Please run the GUI.pyw executable packaged with the Wesnoth installation.").format(
-                          "translations")
-                      )
+            showerror(
+                _("Error"),
+                # TRANSLATORS: {0} is "translations", the directory where compiled translation files (.mo) are stored.
```

Comment:
Huh? Why is format being used at all if it's just a constant string being substituted in?

Reply:
Question for @Slayer95 in #7627? But if I had to guess, I'd say it's because we want the word 'translations' not to be translated. The 'translations' string is the directory name which remains fixed.

Reply:
At the very least, your highlighting of this string made me realise the quote characters needed correcting so thanks for that.